.gitignore

```diff
@@ -45,3 +45,4 @@ testem.log
 .DS_Store
 Thumbs.db
 package-lock.json
+/assets/web
```

Comment:
Try to leave an empty line at end of file wherever you find this symbol

Reply:
Ok OMW